jsonassert-assertj-java8/build.gradle

```diff
@@ -1,4 +1,5 @@
 dependencies {
-	compile project(':jsonassert')
-	compile 'org.assertj:assertj-core:3.3.0'
+	implementation project(':jsonassert')
+	implementation "org.assertj:assertj-core:${assertj8Version}"
+	compile "com.jayway.jsonpath:json-path:${jsonPathVersion}"
```

Comment:
It looks like we are not shading the 7/8 versions?

Reply:
Yeah cause there was no such requirement until now. I don't even know if anyone is using this 🤷‍♀️